arches/app/models/models.py

```diff
@@ -99,11 +81,11 @@ def __init__(self, *args, **kwargs):
         if isinstance(self.cardid, str):
             self.cardid = uuid.UUID(self.cardid)
 
-    def save(self, *args, **kwargs):
+    def save(self, **kwargs):
```

Comment:
Positional arguments to `save()` were [deprecated](https://docs.djangoproject.com/en/5.1/releases/5.1/#id2) in 5.1 and will raise errors in 6.